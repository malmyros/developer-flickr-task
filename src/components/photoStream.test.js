import React from 'react';
import { shallow } from 'enzyme';
import PhotoStream from './photoStream';

describe('photostream', () => {
  it('should render without crashing', () => {
    const mockedCallback = jest.fn();

    const component = shallow(
      <PhotoStream title={'title'}
                   description={'description'}
                   countComments={10}
                   countTotal={3}
                   countViews={4}
                   onLoadPhotosRequest={mockedCallback}
                   onLoadPhotosSuccess={mockedCallback}
                   onLoadPhotosFailure={mockedCallback}
                   onUpdateGallery={mockedCallback}/>,
    );
    expect(component).toMatchSnapshot();
  });
});
