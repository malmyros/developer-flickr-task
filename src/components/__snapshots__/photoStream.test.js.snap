// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`photostream should render without crashing 1`] = `
<section
  className="album py-5 bg-light"
>
  <div
    className="container"
  >
    <div
      className="row"
    >
      <div
        className="col-12 text-center"
      >
        <h1
          className="bd-title"
        >
          title
        </h1>
        <p>
          description
        </p>
        <GalleryStats
          countComments={10}
          countTotal={3}
          countViews={4}
        />
      </div>
    </div>
    <hr />
    <Connect(ChangeViewType) />
    <Connect(Photos) />
  </div>
</section>
`;
